consensus/XDPoS/engines/engine_v2/engine.go

```diff
@@ -267,6 +267,10 @@ func (x *XDPoS_v2) YourTurn(chain consensus.ChainReader, parent *types.Header, s
 
 	waitedTime := time.Now().Unix() - parent.Time.Int64()
 	_, parentRound, _, err := x.getExtraFields(parent)
+	if err != nil {
+		log.Warn("[Yourturn] Error while getExtraFields", "parent", parent.Number, "error", err)
```

Comment:
`"error", err.Error()` need to convert it into string i believe

Reply:
In fact, golang default `error` type, if it require to be String(), then it have predefine rule to convert `err` into `err.Error()`
<img width="341" alt="image" src="https://github.com/XinFinOrg/XDC-Subnet/assets/17850890/b2794d23-237e-4161-b0b7-d373bda70e1d">